### DISCUSSION TITLE
How to troubleshoot videos not playing in Firefox?

### DISCUSSION THREAD

**Discussion by Ailbeenlynd** [sandboxie-plus/Sandboxie#4232] [Q&A Feedback]
I am running an older version of Sandboxie Classic (5.50.2). Firefox does not play videos (any video on any website) when sandboxed. This also happens with Sophos' last version 5.33.3. The same OS + Sandboxie setup works inside Hyper-V though. Edge also doesn't launch, but launches inside the Hyper-V VM with the same setup. AMD GPU. Default Sandboxie.ini configs.

Any tips on what I can open/relax/allow in Sandboxie.ini to troubleshoot the not playing video issue? I uninstalled other security software and same result, video plays unsandboxed and will not start when sandboxed.

**Comment by offhub** [comment]
Please try with the [latest release](https://github.com/sandboxie-plus/Sandboxie/releases/latest) version. Anything other than HEVC should work fine with Firefox without any additional settings.

**Reply by Ailbeenlynd** [reply]
Trying with the latest is the obvious path, but I'd like to try tweaking Sandboxie.ini first. Version 5.50.2 is the last version that does not use driver enforced signature verification on the executables. This allows me to compile my own.

I'd like to relax all sandboxing restrictions in Sandboxie.ini and gradually tighten each one to see if I can narrow the issue down. What are some of the settings I can try? e.g., OpenIpcPath=*, OpenPipePath=*, OpenKeyPath=*, OpenWinClass=* etc, some do not take wildcards?

**Reply by Ailbeenlynd** [reply]
For example, I've tried the following to no avail:
OpenClsid={A4######-####-####-####-001185######}
OpenClsid={DC######-####-####-####-199FDB######}
OpenKeyPath=*
OpenKeyPath=firefox.exe,*
OpenFilePath=*
OpenPipePath=*
OpenPipePath=\Device\*
OpenPipePath=\*
OpenIpcPath=\Sessions\*
OpenIpcPath=*\BaseNamedObjects\*
OpenIpcPath=\RPC Control\*
OpenIpcPath=\SBIE_DummyJob_firefox.exe_1
OpenIpcPath=\KernelObjects\*
OpenIpcPath=\KnownDlls\*
OpenIpcPath=\RPC Control\*
OpenIpcPath=\Security\*
OpenIpcPath=\ThemeApiPort*
OpenWinClass=*

Resource Access Monitor shows all "O"

**Reply by Ailbeenlynd** [reply]
Firefox 32-bit works, will use it as a workaround for now
